protokube/tests/integration/build_etcd_manifest/integration_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"io/ioutil"
+	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/diff"
+	"path"
+	"testing"
+
+	"strings"
+	"k8s.io/kops/protokube/pkg/protokube"
+
+	_ "k8s.io/kubernetes/pkg/api/install"
+	"strconv"
+)
+
+func TestBuildEtcdManifest(t *testing.T) {
+	runTest(t, "main")
+}
+
+func runTest(t *testing.T, srcDir string) {
```

Comment:
There's no special code path for single master, so I'd say no.  Once we add one, yes :-)